read-json.js

```diff
@@ -228,7 +229,7 @@ function readme_ (file, data, rm, cb) {
     // maybe not readable, or something.
     if (er) return cb()
     data.readme = rm
-    data.readmeFilename = rmfn
+    if (!data.readmeFilename) data.readmeFilename = rmfn
```

Comment:
Intent here is to avoid overwriting the `data.readmeFilename` field if it already exists in `package.json`